### PR TITLE
IDVA3-1839 Feature/update start screen

### PR DESCRIPTION
**Pull request by JasSmith-ch**
- updated the start screen
- updated the json file for english and welsh translation

### CODE REVIEW

File: locales/en/start.json [hunk 1]
```diff
@@ -1,40 +1,13 @@
...
-    "start_when_rle_heading": "Relevant legal entities",
-    "start_when_rle_before_date_1": "If the RLE was added to the Companies House register before ",
-    "start_when_rle_before_date_2": ", you must provide its relevant officer's details within 28 days of company's confirmation statement date.",
-    "start_when_rle_after_date_1": "If the RLE was added after ",
-    "start_when_rle_after_date_2": ", you must provide its relevant officer's details within 28 days of the date Companies House accepted the RLE notification filing.",
-    "start_when_inset": "You can find out exactly when you need to provide verification details for a company's PSCs or RLEs by signing into this service and entering the company number.",
+    "start_when_to_verify_heading": "When you need to provide verification details",
+    "start_when_to_verify_description1": "You must wait until the company has reached its confirmation statement date before you can provide verification details for its PSCs or RLEs. You will not be able to do it early",
```
Comment (eedwards0) [inline locales/en/start.json:7]: Minor - add full stop at end

File: locales/en/start.json [hunk 2]
```diff
@@ -46,5 +19,10 @@
     "start_before_required_description": "If you're providing verification details for a relevant officer, you'll need their full name and date of birth.",
     "start_personal_code_details_main_link": "What is a Companies House personal code?",
     "start_personal_code_details_description": "This is an 11 character code that is given to a person after they have verified their identity with Companies House.",
+    "start_providing_details_for_rle": "If you're providing verification details for a relevant officer",
+    "start_deatils_for_rle_description1": "You'll also need their full name and date of birth.",
```
Comment (eedwards0) [inline locales/en/start.json:23]: rename "deatils" to "details"

File: locales/en/start.json [hunk 3]
```diff
@@ -1,40 +1,13 @@
...
+    "start_when_to_verify_heading": "When you need to provide verification details",
+    "start_when_to_verify_description1": "You must wait until the company has reached its confirmation statement date before you can provide verification details for its PSCs or RLEs. You will not be able to do it early",
+    "start_when_to_verify_description2": "The deadline for individual PSCs is 14 days after the confirmation statement date, and RLEs have 28 days.",
+    "start_when_to_verify_description3": "You can see the deadline dates for a company's PSCs and RLEs by signing in to this service.",
+    "start_warning_text": "If a PSC or RLE does not comply with identity verification requirements, they will be committing an offence and may have to pay a financial penalty or fine. We may also display an annotation against their name on the public register.",
     "start_before_heading": "Before you start",
     "start_before_verify_description": "The PSC or relevant officer must",
     "start_before_verify_link": "verify their identity",
```
Comment (eedwards0) [inline locales/en/start.json:13]: May be clearer to describe as "start_verify_link_text"

File: locales/en/start.json [hunk 4]
```diff
@@ -1,40 +1,13 @@
...
-    "start_when_inset": "You can find out exactly when you need to provide verification details for a company's PSCs or RLEs by signing into this service and entering the company number.",
+    "start_when_to_verify_heading": "When you need to provide verification details",
+    "start_when_to_verify_description1": "You must wait until the company has reached its confirmation statement date before you can provide verification details for its PSCs or RLEs. You will not be able to do it early",
+    "start_when_to_verify_description2": "The deadline for individual PSCs is 14 days after the confirmation statement date, and RLEs have 28 days.",
+    "start_when_to_verify_description3": "You can see the deadline dates for a company's PSCs and RLEs by signing in to this service.",
+    "start_warning_text": "If a PSC or RLE does not comply with identity verification requirements, they will be committing an offence and may have to pay a financial penalty or fine. We may also display an annotation against their name on the public register.",
     "start_before_heading": "Before you start",
     "start_before_verify_description": "The PSC or relevant officer must",
```
Comment (eedwards0) [inline locales/en/start.json:12]:
```suggestion
    "start_before_verify_link": "The PSC or relevant officer must",
```

File: locales/en/start.json
Comment (eedwards0) [inline locales/en/start.json:1]: May be clearer to rename to "start_after_verify_link"

Comment (eedwards0) [inline locales/en/start.json:1]: Line 14: May be clearer to rename "start_before_verify_description2" to "start_after_verify_link

**Review by eedwards0 — CHANGES_REQUESTED** [review]
Very minor text and some of the JSON fields may be clearer if renamed, as this page has had a lot of change. Also, there is an extra white space at the end of the "verify their identity" link before the comma.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: locales/en/start.json [hunk 5]
```diff
@@ -1,42 +1,15 @@
...
+    "start_when_to_verify_heading": "When you need to provide verification details",
+    "start_when_to_verify_description1": "You must wait until the company has reached its confirmation statement date before you can provide verification details for its PSCs or RLEs. You will not be able to do it early.",
+    "start_when_to_verify_description2": "The deadline for individual PSCs is 14 days after the confirmation statement date, and RLEs have 28 days.",
+    "start_when_to_verify_description3": "You can see the deadline dates for a company's PSCs and RLEs by signing in to this service.",
+    "start_warning_text": "If a PSC or RLE does not comply with identity verification requirements, they will be committing an offence and may have to pay a financial penalty or fine. We may also display an annotation against their name on the public register.",
     "start_before_heading": "Before you start",
-    "start_before_verify_description": "The PSC or relevant officer must",
+   "start_before_verify_link_text": "The PSC or relevant officer must",
```
Comment (eedwards0) [inline locales/en/start.json:12]: Rename "start_before_verify_link_text" to "start_before_verify_link"

File: src/views/router_views/start/start.njk [hunk 6]
```diff
@@ -3,81 +3,31 @@
...
+     <p class="govuk-body">{{ i18n.start_when_to_verify_description2 }}</p>
+     <p class="govuk-body">{{ i18n.start_when_to_verify_description3 }}</p>
+
+     {{ govukWarningText({
+        text: i18n.start_warning_text
     }) }}
 
-    <h2 class="govuk-heading-m">{{ i18n.start_before_heading }}</h2>
+        <h2 class="govuk-heading-m">{{ i18n.start_before_heading }}</h2>
     <p class="govuk-body">{{ i18n.start_before_verify_description }}
```
Comment (eedwards0) [inline src/views/router_views/start/start.njk:31]: Please update lines 31 to 35 as follows:

    <p class="govuk-body">{{ i18n.start_before_verify_link }}
        <a href="{{ [chsUrl, verifyIdentityLink] | join  }}" title="{{ i18n.start_verify_link_text }}" class="govuk-link" data-event-id="before-verify-link">
            {{ i18n.start_verify_link_text }}
        </a>
        {{i18n.start_after_verify_link}}</p>

**Review by eedwards0 — CHANGES_REQUESTED** [review]
Some tweaks needed - suggested changes in comments.
Also, can you take a look at removing the extra white space after the 'verify their identity' link under the 'Before you start' section?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/router_views/start/start.njk [hunk 7]
```diff
@@ -3,87 +3,39 @@
...
+        {{ i18n.start_before_verify_link }}
+        <a href="{{ chsUrl }}" title="{{ i18n.start_service_ch_link }}" title="{{ i18n.start_after_verify_link }}" class="govuk-link" data-event-id="when-to-verify-link">
+            {{ i18n.start_after_verify_link_text }}
+            {{ i18n.start_verify_link_text }}
         </a>
-        {{i18n.start_before_verify_description2}}</p>
-    <p class="govuk-body">{{ i18n.start_before_list }}</p>
+        {{ i18n.start_after_verify_link }}
+        </p>
     <ul class="govuk-list govuk-list--bullet">
```
Comment (eedwards0) [inline src/views/router_views/start/start.njk:39]: Please add in: 
<p class="govuk-body">{{ i18n.start_before_list }}</p> 
after the verify link text, which ends on line 38, and before the bulleted list on line 39.

**Review by eedwards0 — CHANGES_REQUESTED** [review]
Just need to add back in lines before the bulleted list - added comment to start.njk

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by eedwards0 — APPROVED** [review]
Looking good :)

**Pull request merged**